Add support for changing user role ARN

jtallinger opened the pull request:
Hi,

My first pull request so please bare with me if this is not done entirely right. 😄 

These changes allow authentication with a common AWS user account and then switch to required role ARN.

Drone yml settings example:
```
user_role_arn: arn:aws:iam::012345678901:role/APP_TEST_QA_SERVICE
access_key:
    from_secret: aws_access_key_id
secret_key:
    from_secret: aws_secret_access_key
```

Issue: https://github.com/josmo/drone-ecs/issues/52

josmo commented on plugin.go:
```diff
@@ -81,7 +83,22 @@ func (p *Plugin) Exec() error {
+		var arnCredentials *credentials.Credentials
+
+		awsConfigArn := aws.Config{Region: aws.String(p.Region)}
+		arnCredentials = stscreds.NewCredentials(sess, p.UserRoleArn, func(p *stscreds.AssumeRoleProvider) {})
```
I think this can just be ```arnCredentials := stscreds.NewCredentials(sess, p.UserRoleArn)``` and you can delete line 92 but if you don't mind trying that out :)

josmo commented:
Thanks a bunch @jtallinger I just have the one comment to try out :) 

also, just an fyi (doesn't matter in this case) but it'll save some headaches (in more active projects) for you if you branch out of master in your fork first and PR from that branch.  That way if there was a lot of development in the project it would be easier to pull master from the upstream and merge back into your branch and into the PR :)

jtallinger commented:
Thanks, that makes sense regarding the checkout, did not think about it. 👍 

I changed those lines and removed some debug printing I added earlier. I also updated the doc with the new parameter.
